resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -131,7 +117,7 @@ params+=('--enterprise-build')
 <% end -%>
 
 # pull in rpm dependencies
-if [[ "$build_os" =~ el-?7 || "$build_os" =~ el-?6 || "$build_os" =~ sles || "$build_os" =~ fedora ]]; then
+if [[ "$os" = 'el' || "$os" = 'sles' || "$os" = 'fedora' ]]; then
```

Comment:
This looks much better! Less for us to have to remember to update in the future.